create-k8s-chained-sessions.sh

```diff
@@ -90,7 +96,8 @@ PARENT_SESSION_NAMES="panorama-k8s-playground panorama-k8s-playground-2 panorama
 
 for session in $PARENT_SESSION_NAMES
 do
-    createLeappSession "$session" "AWSAdministratorAccess" "eks-admin-1.24"
-    createLeappSession "$session" "PanoramaK8sEngineeringDefault" "panorama-dev-writer-1.24"
-    createLeappSession "$session" "PanoramaK8sEngineeringDefault" "panorama-dev-reader-1.24"
+    createLeappSession "$session" "AWSAdministratorAccess" "eks" "admin" "1.24"
+    createLeappSession "$session" "PanoramaK8sEngineeringDefault" "panorama" "dev-writer" "1.24"
+    createLeappSession "$session" "PanoramaK8sEngineeringDefault" "panorama" "dev-reader" "1.24"
+    createLeappSession "$session" "PanoramaK8sDSAR" "panorama" "dsar-tester" "1.24"
```

Comment:
@Pat-Ayres I realized I forgot to update `dsar-tester` to `data-scientist-tester` (or just `data-scientist` if you prefer). I'll hold off on the change and wait for any other code review feedback to minimize back and forth.

Reply:
Update: I haven't committed it here yet, but in my testing, I'm using `data-science-tester`, so if no objections to that, I will just stick with that name moving forward.